pkg/relay/ethereum/events.go

```diff
@@ -81,6 +81,9 @@ func (chain *Chain) findSentPackets(ctx core.QueryContext, fromHeight uint64) (c
 		if err != nil {
 			return nil, fmt.Errorf("failed to parse SendPacket event: err=%v, log=%v", err, log)
 		}
+		if sendPacket.SourceChannel != chain.Path().ChannelID {
+			continue
+		}
```

Comment:
As a result of the discussion with @bluele, it was decided that port id should also be compared.

Reply:
Added check `portID`

Reply:
@dongrie replace `&&` with `||` ...

Reply:
orz.... sorry, Fixed
https://github.com/datachainlab/ethereum-ibc-relay-chain/pull/38/commits/6eafa2e553de62ee44de1b2a460ee3e1f547b6b6